zebrad/tests/acceptance.rs

```diff
@@ -2157,3 +2157,13 @@ async fn sending_transactions_using_lightwalletd() -> Result<()> {
 async fn lightwalletd_wallet_grpc_tests() -> Result<()> {
     common::lightwalletd::wallet_grpc_test::run().await
 }
+
+/// Test successful submitblock rpc call
+///
+/// See [`common::getblocktemplate`] for more information.
+#[tokio::test]
+#[ignore]
+#[cfg(feature = "getblocktemplate-rpcs")]
+async fn submit_block() -> Result<()> {
```

Comment:
I think this test need a section in the main acceptance documentation at the top, similar to lightwalletd https://github.com/ZcashFoundation/zebra/blob/main/zebrad/tests/acceptance.rs#L56 explaining what environment variables and other things you need to run this test.

Reply:
I think we should run this in our CI in the context of https://github.com/ZcashFoundation/zebra/issues/5527, not sure why it was closed as not planned.

Reply:
I should've explained, I just left a comment in #5527 now. I'm going to try to update the test so that it can be run in the coverage CI and will re-open the issue if it's still necessary.

Reply:
> I think this test need a section in the main acceptance documentation at the top, similar to lightwalletd https://github.com/ZcashFoundation/zebra/blob/main/zebrad/tests/acceptance.rs#L56 explaining what environment variables and other things you need to run this test.

Added in [`bcc6c28`](https://github.com/ZcashFoundation/zebra/pull/5526/commits/bcc6c28118559f0a48f07837ae88023c37361902)